units/user/xdg-desktop-autostart.target

```diff
@@ -10,4 +10,5 @@
 [Unit]
 Description=Startup of XDG autostart applications
 Documentation=man:systemd.special(7)
+RefuseManualStart=yes
```

Comment:
can you elaborate?

Reply:
Well, my thought here is that the unit should be pulled in by the desktop, in the same way that `graphical-session.target` handles it.

Reply:
ok